types/recharts/index.d.ts

```diff
@@ -417,6 +417,7 @@ export interface LegendPayload {
     id: any;
     type: LegendType;
     color?: string;
+    payload?: Record<"strokeDasharray", string>;
```

Comment:
this doesn't make sense to me, I apologize.  Do you mean to write:

```ts
  payload?: {
    strokeDasharray: string,
  }
```

Reply:
you're right, your way is more direct